### PR TITLE
Add r-ffield.

### PR DESCRIPTION
**Pull request by cbrueffer**
This is a dependency for bioconda/bioconductor-genvisr.

### CODE REVIEW

**Comment by jdblischak** [comment]
I restarted the Travis and Appveyor builds now that conda-smithy 2.4.3 has been released.

**Comment by bgruening** [comment]
restarted again

**Comment by jdblischak** [comment]
And I restarted AppVeyor. It appeared pending, but had actually been canceled.

File: recipes/r-ffield/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    - r-gridextra
+
+test:
+  commands:
+    - $R -e "library('FField')"  # [not win]
+    - "\"%R%\" -e \"library('FField')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=FField
+  license: GPL-3
```
Comment (bgruening) [inline recipes/r-ffield/meta.yaml]: can you please add a license_file as well like here: https://github.com/bgruening/conda_r_skeleton_helper/blob/master/run.sh#L20

Comment (cbrueffer) [reply recipes/r-ffield/meta.yaml]: Done, thanks for the hint!

**Comment by bgruening** [comment]
Thanks @cbrueffer!

**Pull request merged**